extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/OpenshiftDeployer.java

```diff
@@ -17,6 +18,8 @@ public class OpenshiftDeployer {
 
     @BuildStep
     public void checkEnvironment(Optional<SelectedKubernetesDeploymentTargetBuildItem> selectedDeploymentTarget,
+            // this is required to ensure the kubernetes client is fully configured when calling the `client.isSupported()`:
+            KubernetesClientOriginalValueDisableDnsResolverBuildItem kubernetesClientConfigured,
```

Comment:
FYI you can also use `@Consume`:

```suggestion
    @BuildStep
    // this is required to ensure the kubernetes client is fully configured when calling the `client.isSupported()`:
    @Consume(KubernetesClientOriginalValueDisableDnsResolverBuildItem.class)
    public void checkEnvironment(Optional<SelectedKubernetesDeploymentTargetBuildItem> selectedDeploymentTarget,
```

... also, you will need to add an import.

Reply:
I've changed the solution, this change does not apply any longer.